### ISSUE TITLE
VGG classifier setting different from Original paper

### ISSUE THREAD

**Issue by lichengunc** [pytorch/vision#92]
One of the dropout layer was "wrongly" inserted.
The original final layers of Caffe Version (https://gist.github.com/ksimonyan/211839e770f7b538e2d8) is:
self.classifier = nn.Sequential(
            nn.Linear(512 * 7 * 7, 4096),
            nn.ReLU(True),
            nn.Dropout(),
            nn.Linear(4096, 4096),
            nn.ReLU(True),
            nn.Dropout()
            nn.Linear(4096, 1000),
        )
This won't make difference when we use model.eval(), but will make discrepancy if we want to finetune VggNet by loading Caffe's parameters.

**Comment by colesbury** [comment]
Thanks for catching this. I'll fix this and retrain the VGG models.
